examples/config_template.json

```diff
@@ -6,12 +6,12 @@
     },
     "select": {
         "alias n°1": [
-            "expression of attribute a of resource type 1",
-            "expression of attribute b of resource type 1",
-            "expression of attribute c of resource type 1"
+            "searchparam of attribute a of resource type 1",
+            "searchparam of attribute b of resource type 1",
+            "searchparam of attribute c of resource type 1"
```

Comment:
The fhirpath expression remains usable, right ? We can use either searchparam or expression there ?

Reply:
It's true! When I made the change I hadn't done the tests yet but it's true that it works! I modified it